angrmanagement/ui/workspace.py

```diff
@@ -2,7 +2,10 @@
 import logging
 from collections import defaultdict
 
-from PySide2.QtCore import Qt
+from PySide2.QtCore import Qt, QSettings
+
+from PySide2.QtWidgets import QSplitter, QMainWindow, QShortcut
```

Comment:
Nitpick: The blank line above this import is unnecessary. All imports from PySide2 should be grouped together.

Reply:
Oops, sorry. I missed this one. Fixed.